cardano-node/src/Cardano/CLI/Run.hs

```diff
@@ -41,58 +41,35 @@ module Cardano.CLI.Run (
   , NewTxFile(..)
```

Comment:
Are all of the `LANGUAGE` and `OPTIONS_GHC` pragmas still required in this module? (`GeneralisedNewtypeDeriving ` and `-Wno-all-missed-specialisations` in particular)

Reply:
`-Wno-all-missed-specialisation` is, ghc complains:
```
src/Cardano/CLI/Run.hs: warning:
    Could not specialise imported function ‘cardano-ledger-0.1.0.0:Cardano.Chain.Genesis.Data.$wreadGenesisData’
      when specialising ‘readGenesisData’
    Probable fix: add INLINABLE pragma on ‘cardano-ledger-0.1.0.0:Cardano.Chain.Genesis.Data.$wreadGenesisData
```
And as for `GeneralizedNewtypeDeriving`:
```
src/Cardano/CLI/Run.hs:141:28: error:
    • Can't make a derived instance of ‘IsString NewTxFile’:
        ‘IsString’ is not a stock derivable class (Eq, Show, etc.)
        Try GeneralizedNewtypeDeriving for GHC's newtype-deriving extension
    • In the newtype declaration for ‘NewTxFile’
    |
141 |   deriving (Eq, Ord, Show, IsString)
```

Reply:
But `{-# OPTIONS_GHC -Wno-partial-fields #-}` needs to go :)

Reply:
Perhaps we can look into whatever fix is required for the `all-missed-specialisations` warning (I haven't looked into it). Probably not worth blocking this PR on.

Reply:
> But `{-# OPTIONS_GHC -Wno-partial-fields #-}` needs to go :)

100%. I see no reason that we should ever suppress this warning.

Reply:
Again, @intricate, @Jimbo4350, this was fixed in https://github.com/input-output-hk/cardano-node/pull/152/files#diff-75f92b9083d544707ede537871a2a640L21

Did we have a clean rebase?